### DISCUSSION TITLE
Share your Lazy dotfiles

### DISCUSSION THREAD

**Discussion by iskeld** [LazyVim/LazyVim#766] [General]
Hi, I'm new to NeoVim (I was using Vim before) and wanted to bootstrap my setup using LazyVim. 
However I'm still unsure where to put which setting (for instance `vim.opt.background = "light"`) or how to organize my config.

So if anyone is willing to share their non-default-LazyVim-based setups I would be grateful (I'm also looking for a config with Rust development features but any non-default config will do).

**Comment by loiccoyle** [comment]
here are my dots: https://github.com/loiccoyle/dots/tree/master/.config/nvim/

For some rust inspiration, checkout this PR (which was closed for some reason): https://github.com/LazyVim/LazyVim/pull/24

**Reply by System3-2** [reply]
nice config you got there man

**Reply by loiccoyle** [reply]
thanks !

**Comment by folke** [comment]
Here you go :) https://github.com/folke/dot/tree/master/nvim

**Comment by dpetka2001** [comment]
Here's mine https://github.com/dpetka2001/dotfiles/tree/main/dot_config/nvim

**Reply by washere** [reply]
Thanks, your dotfiles are the only one that did not give errors for me + it is a big bundle & nice ones too.
I hope more Lazyvim  users share their dotfiles here.  :+1:

**Reply by washere** [reply]
NullChilly theme: Also whatever your configs, for best coder friendly dark colorscheme, a Catppuccin slight mod apparently:
Save this raw text as nullchilly.lua or whatever .lua & copy to your plugins folder:

https://raw.githubusercontent.com/nullchilly/CatNvim/bea18b53958627aff3051c22591a4f5771a882c8/lua/plugins/colorscheme.lua

[![nullchilly](https://user-images.githubusercontent.com/56817415/197395733-6f9f254c-57b6-4843-9ac8-1ce0302faf9e.png)]

best to delete other colorscheme lua files in plugins folder but should override most anyway.

**Comment by amaanq** [comment]
Mine are @ https://github.com/amaanq/nvim-config

Check out plugins/extras/lang/rust for some inspiration to derive a feasible Rust setup :slightly_smiling_face:

**Reply by ditsuke** [reply]
I wanted to share that I've taken a lot of inspiration from your setup for my own. Have it cloned and ready to go even now. Thank you :)

**Reply by amaanq** [reply]
Glad to hear!

**Comment by cutbypham** [comment]
- Neovim config: https://github.com/licitfree/dotfiles/releases/tag/v4
- Front-end dev, light theme, tmux user

**Reply by timjackleus** [reply]
Lots of good stuff thanks for sharing!

**Comment by hopezh** [comment]
Here's my work-in-progress NeoVim config: [OatVim](https://github.com/hopezh/OatVim), using lazy.nvim and learning from LazyVim heavily. 

![238320133-839849d1-3b7d-4641-96b9-065dadf8cf9b](https://github.com/LazyVim/LazyVim/assets/23435810/5fd0132b-6d2c-4a1e-b591-3d3f265189fc)

![238321270-7a06b380-4453-4da6-b385-d555244daf35](https://github.com/LazyVim/LazyVim/assets/23435810/93f0a5f1-b6db-4147-ab9c-11bf9661b8ac)

**Comment by ditsuke** [comment]
Here's mine: https://github.com/ditsuke/nvim-config

**Comment by fredrikaverpil** [comment]
https://github.com/fredrikaverpil/dotfiles/tree/main/nvim-lazyvim/lua 😄

**Comment by utkarshgupta137** [comment]
I think my setup is finally stabilized a bit: https://github.com/utkarshgupta137/nvim-lazy

**Comment by ZayRTun** [comment]
For Laravel/PHP check out mine https://github.com/ZayRTun/Lazyvim-for-Laravel

**Comment by rudenkornk** [comment]
Ansible-based CI-tested configuration. Main setup is `Ubuntu + tmux + neovim + lazy`
https://github.com/rudenkornk/dotfiles
https://github.com/rudenkornk/dotfiles/tree/main/roles/neovim/files/lazy

**Comment by Buer-Nahida** [comment]
Here's mine: https://github.com/SR-MyStar/MyNeovimDots
![2023-12-14-01-50-55-window](https://github.com/LazyVim/LazyVim/assets/139795416/fd93194e-0012-44b2-b30e-7de81f2e4285)

**Comment by m0lson84** [comment]
Here's my dotfiles managed by [chezmoi.io](https://www.chezmoi.io/). Daily driving Zellij + Neovim on MacOS/WSL2.
https://github.com/m0lson84/dotfiles

<img width="1168" alt="image" src="https://github.com/LazyVim/LazyVim/assets/48652773/972230fe-2161-431d-b49a-93dbc1bcb2ae">

**Comment by rubiin** [comment]
Here is mine: https://github.com/rubiin/init.lua

**Comment by schoblaska** [comment]
My config with Diffview, Telescope, oil, and bqf integration, plus some other optimizations for reviewing and exploring code and git histories: https://github.com/schoblaska/neovim

Layout configs are calm and simple and use all available space

<img width="3022" height="1732" alt="image" src="https://github.com/user-attachments/assets/8e4e2186-48f2-4f45-ba1e-537752514e6c" />

**Comment by juanlopez4691** [comment]
Here is mine: https://github.com/juanlopez4691/nvim-lazyvim
<img width="5344" height="3044" alt="Screenshot 2026-01-26 at 12 30 00" src="https://github.com/user-attachments/assets/2ccdbdad-464f-4fcf-bad9-53c2af16b7ad" />
